recipe/install-r-xgboost.bat

```diff
@@ -2,10 +2,14 @@
 mkdir "%SRC_DIR%"\build-r
 pushd "%SRC_DIR%"\build-r
 
+set PATH=%LIBRARY_PREFIX%\bin;%PATH%
```

Comment:
Is this still needed? Would think `%LIBRARY_PREFIX%\bin` is already in `%PATH%`

Reply:
Yes, this fix is needed because `%BUILD_PREFIX%\bin` comes before `%LIBRARY_PREFIX%\bin%, and this breaks the build. So prepending `%LIBRARY_PREFIX%\bin` seems to be needed to guarantee that for executables found in both prefixes, the one in `%LIBRARY_PREFIX%\bin` is preferred.

Reply:
Having said that, it is worth trying to remove it now and see if it's still needed.

Reply:
Sounds good let's try removing

If it is still needed, including a code comment along the lines of what you have written would be helpful for future readers